export const rethrow = (err: unknown) => {
  throw err;
};
